rns_shell/compositor/layers/Layer.cpp

```diff
@@ -139,14 +139,9 @@ void Layer::preRoll(PaintContext& context, bool forceLayout) {
         absFrame_ = SkIRect::MakeXYWH(mapRect.x(), mapRect.y(), mapRect.width(), mapRect.height());
         SkIRect newBounds = absFrame_;
         frameBounds_ = frame_;
-        if(shadowFilter) {
+        if(componentShadow.imageFilter) {
```

Comment:
condition needs to be Either maskFilter or ShadowFilter exist

componentShadow.imageFilter || componentShadow.maskFilter